src/main/fc/fc_msp.c

```diff
@@ -1032,6 +1032,64 @@ static bool mspFcProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst, sbuf_t *src, msp
         sbufWriteU8(dst, masterConfig.gyroSync);
         break;
 
+    case MSP_FILTER_CONFIG :
```

Comment:
We also have `currentProfile->pidProfile.acc_soft_lpf_hz` in INAV. I wonder if we should add it to MSP_FILTER_CONFIG as well. This will break compatibility with BF though.